src/Modules/Manage/Dnn.PersonaBar.Users/Users.Web/src/utils/PasswordStrength.js

```diff
@@ -0,0 +1,63 @@
+const passwordStrength = (password, options) => {
```

Comment:
It's nice to have here but my concern is that this could get out of sync with back end settings. Am I right?